RecoHGCal/TICL/plugins/TrackstersMergeProducer.cc

```diff
@@ -499,11 +507,12 @@ void TrackstersMergeProducer::produce(edm::Event &evt, const edm::EventSetup &es
       usedSeeds[s.index] = true;
     }
   }
+
   // for all general tracks (high purity, pt > 1), check if they have been used: if not, promote them as charged hadrons
   for (unsigned i = 0; i < tracks.size(); ++i) {
     auto const &track = tracks[i];
-    if (track.pt() > 1.f and track.quality(reco::TrackBase::highPurity) and track.missingOuterHits() < 5 and
-        std::abs(track.outerEta()) > 1.48 and std::abs(track.outerEta()) < 3.0 and usedSeeds[i] == false) {
+    if (track.pt() > track_min_pt_ and track.quality(reco::TrackBase::highPurity) and track_max_missing_outerhits_ < 5 and
```

Comment:
track_max_missing_outerhits_ < 5 
should be 
track.missingOuterHits() < track_max_missing_outerhits_

Reply:
thanks @mariadalfonso , you're right, fixed